config/config.go

```diff
@@ -109,6 +109,17 @@ var (
 	DefaultMarathonSDConfig = MarathonSDConfig{
 		RefreshInterval: Duration(30 * time.Second),
 	}
+
+	// The default Kubernetes SD configuration
+	DefaultKubernetesSDConfig = KubernetesSDConfig{
+		NodeMetricsPort: 10255,
+		ScrapeKey:       "prometheus.io/scrape",
```

Comment:
> What is the difference in doing this via relabelling or via SD config?

We want to generally avoid every SD mechanism having to reimplement the same things again and again. We also want to avoid SD mechanisms that make assumptions about all deployments that aren't actually true.

Thus we have relabelling, as it turns out almost everything that SD configs want to do can be handled by it and that most of those things weren't globally applicable in the first place. The most a SD should be providing is a host, port and a pile of metadata.

> This isn't specific to my setup, but a pre-configured key to be used on annotations on service to enable scraping of service endpoints. 

Where is this pre-configured key coming from? Is it guaranteed to be used this way for all Kubernetes setups in existence? If it is, why isn't it following the existing naming scheme for these labels? Where was the discussion to create this convention?

> I'm totally unsure how relabelling could be used in this case as this annotation actually prevents target groups from being created if the key isn't present & truthy.

The `drop` and `keep` relabel actions allow for limiting what targets are created.